resource/css/_search.scss

```diff
@@ -87,15 +92,13 @@
 }
 
 .search-suggest {
-  max-width: 500px;
+  width: 100%;
```

Comment:
If here to fix value to `100%`, small width device displays really narrow area.
So, should we specify `min-width` or the style between `md` and `lg`?

For example, below is around 800px width window.

![image](https://user-images.githubusercontent.com/29064/63235318-e32e4a00-c273-11e9-9158-c1a918cd6558.png)